include/deal.II/grid/grid_tools.h

```diff
@@ -797,13 +797,17 @@ namespace GridTools
     const std::vector<std::vector<BoundingBox<spacedim>>> &global_bboxes);
 
   /**
-   * Return a map of index:Point<spacedim>, containing the used vertices of the
-   * given `container`. The key of the returned map is the global index in the
-   * triangulation. The used vertices are obtained by looping over all cells,
+   * Return a map `vertex index -> Point<spacedim>`, containing the used
```

Comment:
I reckon that there shouldn't be a comma after `vertex index -> Point<spacedim>`